src/Call.tsx

```diff
@@ -895,8 +895,8 @@ export class Call extends EventEmitter {
    * ```
    *
    * @param content - The message content
-   * @param contentType - The MIME type for the message. See {@link (CallMessage:namespace).ContentType}.
-   * @param messageType - The message type. See {@link (CallMessage:namespace).MessageType}.
+   * @param contentType - The MIME type for the message.
```

Comment:
Can we add `application/json` as the only supported type for now and it's the default?